pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -167,9 +171,23 @@ type AlertmanagerReference struct {
 type ThanosQuerierReference struct {
 	// Url of the ThanosQuerier to proxy to.
 	//
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:MinLength:=1
-	Url string `json:"url"`
+	// +kubebuilder:validation:Optional
+	Url string `json:"url,omitempty"`
```

Comment:
same here

Reply:
Updated in the latest commit to require thanosQuerier and alertmanager URL